docs/installation_guide.md

```diff
@@ -147,6 +147,20 @@ To install Harbor with Notary service, add a parameter when you run ```install.s
 More information about Notary and Docker Content Trust, please refer to Docker's documentation: 
 https://docs.docker.com/engine/security/trust/content_trust/
 
+##### Installation with Clair
+To install Harbor with Clair service, add a parameter when you run ```install.sh```:
+```sh
+    $ sudo ./install.sh --with-clair
+```
+
+For more information about Clair, please refer to Clair's documentation: 
+https://coreos.com/clair/docs/2.0.1/
+
+**Note**: If you want to install both Notary and Clair, you can specify both parameters in the same command:
```

Comment:
you can specify both parameters in the same command:
==>
you must specify both parameters in the same command: